src/components/ModelTableList/StatusGroup/StatusGroup.test.tsx

```diff
@@ -177,50 +164,19 @@ describe("StatusGroup", () => {
     );
   });
 
-  it("displays links to blocked apps and units", async () => {
-    vi.useFakeTimers();
+  it("displays links to blocked apps", async () => {
```

Comment:
I think we still need to test that these links get rendered, however, I've removed the bit regarding the Tooltip testing, as the tests for it are present in `WarningMessage.test.tsx`.